src/corePlugins.js

```diff
@@ -848,7 +848,7 @@ export let corePlugins = {
   ),
   rotate: createUtilityPlugin(
     'rotate',
-    [['rotate', [['@defaults transform', {}], '--tw-rotate', ['transform', cssTransformValue]]]],
+    [['rotate', ['--tw-rotate', ['transform', 'rotate(var(--tw-rotate))']]]],
```

Comment:
I know that this goes against the pattern of the other transform utilities, but I didn't want to do all of them at once and make this a big PR, especially if this pattern gets rejected.